src/test/java/core/basesyntax/dao/FruitStorageDaoImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.db.FruitStorage;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitStorageDaoImplTest {
```

Comment:
Actually, there is no need to test Dao layer, because there is no business logic